### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by Sohambasu07**
Changes made:

* Clipped PD1 surrogates' results by metric's max bounds
* Changed minimum fidelity values of MFHartmann, JAHSBench, PD1 imagenet and PD1 cifar100 to 1
* Corrected surrogate names of PD1 cifar100 and PD1 translate_wmt
* Changed `jahs_bench.Benchmark(task=self.task_id) to (task=task)`

### CODE REVIEW

File: src/mfpbench/jahs/benchmark.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ class JAHSResult(Result[JAHSConfig, int]):
 
 class JAHSBenchmark(Benchmark[JAHSConfig, JAHSResult, int], ABC):
     JAHS_FIDELITY_NAME: ClassVar[str] = "epoch"
-    JAHS_FIDELITY_RANGE: ClassVar[tuple[int, int, int]] = (3, 200, 1)
+    JAHS_FIDELITY_RANGE: ClassVar[tuple[int, int, int]] = (1, 200, 1)
```
Comment (eddiebergman) [inline src/mfpbench/jahs/benchmark.py:80]: This was previously the case but we had to make some changes for NePS related stuff. I'm happy this is set back to this :) Thank you!

File: src/mfpbench/jahs/benchmark.py [hunk 2]
```diff
@@ -198,7 +198,7 @@ def bench(self) -> jahs_bench.Benchmark:
                 )
 
             self._bench = jahs_bench.Benchmark(
-                task=self.task_id,
+                task=task,
```
Comment (eddiebergman) [inline src/mfpbench/jahs/benchmark.py:201]: Nice spot

File: src/mfpbench/pd1/benchmark.py [hunk 3]
```diff
@@ -201,7 +201,8 @@ def _results_for(
         features = xs.columns
         for metric, surrogate in self.surrogates.items():
             # We clip as sometimes the surrogate produces negative values
-            xs[metric] = surrogate.predict(xs[features]).clip(min=0)
+            bounds = self.Result.metric_defs[metric].bounds
+            xs[metric] = surrogate.predict(xs[features]).clip(min=0, max=(bounds[1]))
```
Comment (eddiebergman) [inline src/mfpbench/pd1/benchmark.py:205]: I guess this could just be `.clip(*bounds)`?

**Review by eddiebergman — APPROVED** [review]
Looks good.

Has this been tested with a clean download?

**Comment by Sohambasu07** [comment]
> Looks good.
> 
> Has this been tested with a clean download?

Hey, yes. I've run all tests and tested that the surrogates return valid values as well

File: src/mfpbench/pd1/benchmark.py (hunk 3 above)
Comment (Sohambasu07) [reply src/mfpbench/pd1/benchmark.py]: Changed that in the next commit

**Review by eddiebergman — APPROVED** [review]

**Pull request merged**